src/service/poc_lora.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+import "data_rate.proto";
+
+enum invalid_participant_type {
+  beaconer = 0;
+  witness = 1;
+}
+
+enum invalid_reason {
+  none = 0;
+  bad_signature = 1;
+  not_asserted = 2;
+  entropy_expired = 3;
+  bad_entropy = 4;
+  invalid_capability = 5;
+  invalid_pubkey = 6;
+  irregular_interval = 7;
+  // TODO: add final values
+}
+
+message lora_ingest_report_v1 {
+  uint64 received_timestamp = 1;
+  oneof report {
+    lora_beacon_report_req_v1 beacon = 2;
+    lora_witness_report_req_v1 witness = 3;
+  }
+}
+
+message lora_beacon_report_req_v1 {
+  bytes beacon_id =
+      1; // likely be a concatenation of local and remote entropy so maybe not
+         // needed if we already include both fields ?
+  bytes pub_key = 2;
+  bytes local_entropy = 3;
+  bytes remote_entropy = 4;
+  bytes data = 5;
+  float frequency = 6;
+  int32 channel = 7;
+  data_rate datarate = 8;
+  int32 tx_power = 9;
+  // Timestamp of beacon report submit
+  uint64 timestamp = 10;
+  bytes signature = 11;
+}
+
+message lora_beacon_report_resp_v1 { string id = 1; }
+
+message lora_witness_report_req_v1 {
+  bytes beacon_id = 1;
+  bytes pub_key = 2;
+  bytes packet = 3;
+  uint64 timestamp = 4;
+  uint32 ts_res = 5;
+  sint32 signal = 6;
+  float snr = 7;
+  float frequency = 8;
+  int32 channel = 9;
+  data_rate datarate = 10;
+  bytes signature = 11;
```

Comment:
We also need to attach location to the witness and beacon reports. So when the verifier is outputting the files to S3, it needs to query the follower and attach location to the witness and beacon reports.

Furthermore, we also need a place for attaching the hex_scale (or tx_scale as its specified in rewards_v2 on core). So that the injector service can just look up the files output from verifier directly and just construct the poc_receipts_v2 transaction.